tests/suites/test_suite_opaque_test_driver.function

```diff
@@ -0,0 +1,643 @@
+                                                          OPQ_BUFSIZE,
+                                                          &out_key_len ) );
+    TEST_ASSERT( PSA_ERROR_BUFFER_TOO_SMALL == opaque_test_driver_import_key(
+                                                          &attr,
+                                                          in_key,
+                                                          16,
+                                                          out_key,
+                                                          27,
+                                                          &out_key_len ) );
+
+    /* Check validity of parameter set. */
+    TEST_ASSERT( PSA_SUCCESS == opaque_test_driver_generate_key(
+                                                          &attr,
+                                                          out_key,
+                                                          OPQ_BUFSIZE,
+                                                          &out_key_len ) );
+    psa_set_key_bits( &attr, 0 );
+    TEST_ASSERT( PSA_ERROR_INVALID_ARGUMENT == opaque_test_driver_generate_key(
+                                                          &attr,
+                                                          out_key,
+                                                          OPQ_BUFSIZE,
+                                                          &out_key_len ) );
+    psa_set_key_bits( &attr, 128 );
+    psa_set_key_type( &attr, PSA_KEY_TYPE_ARC4 );
+    TEST_ASSERT( PSA_ERROR_NOT_SUPPORTED == opaque_test_driver_generate_key(
+                                                          &attr,
+                                                          out_key,
+                                                          OPQ_BUFSIZE,
+                                                          &out_key_len ) );
+    psa_set_key_type( &attr, PSA_KEY_TYPE_AES );
+    TEST_ASSERT( PSA_ERROR_BUFFER_TOO_SMALL == opaque_test_driver_generate_key(
+                                                          &attr,
+                                                          out_key,
+                                                          27,
+                                                          &out_key_len ) );
+    psa_set_key_bits( &attr, 129 );
+    TEST_ASSERT( PSA_ERROR_NOT_SUPPORTED == opaque_test_driver_generate_key(
+                                                          &attr,
+                                                          out_key,
+                                                          OPQ_BUFSIZE,
+                                                          &out_key_len ) );
+    psa_set_key_bits( &attr, 128 );
+
+    /* Check validity of parameter set. */
+    TEST_ASSERT( PSA_SUCCESS == opaque_test_driver_export_public_key(
+                                    &attr,
+                                    (uint8_t *) "OPQTDKHEADERHello world 1234",
```

Comment:
Why suddenly start to hard-code strings here? Please define and use a `const uint8_t[]`, and use `sizeof` rather than hard-code its size.